styles/theme.ts

```diff
@@ -27,6 +27,10 @@ const config: ThemeOverride = {
         default: "gray.100",
         _dark: "gray.900",
       },
+      tertiaryBackgroundBorder: {
+        default: "gray.200",
+        _dark: "gray.600",
```

Comment:
IMO the border for the description in the dark theme looks a bit harsh:

![harsh-border-for-description](https://user-images.githubusercontent.com/22376627/155054474-bf1db86a-d1e1-4de6-992a-b5be8d34ba8a.png)

What are your opinions on adjusting it to the colour of the horizontal rule?

Reply:
Good call. I wasn't really happy with the dark mode colours for this either so I updated the border colour to match the divider as you suggested and lightened it a bit.